### PR TITLE
Junit for copy-paste action on selecting title blocks

### PR DESCRIPTION
**Pull request by georgiana-ecobici**

### CODE REVIEW

**Comment by pdulth** [comment]
@claudiasima-ro can you look, it seems there is an issue with your eclipse account
https://github.com/eclipse/capella/wiki/How-to-contribute#legal-stuff

(your mail in Eclipse account shall be the same than Github account, Github ID shall be registered in Eclipse account and Signed-Off commits shall have same mail.)

https://accounts.eclipse.org/legal/eca/validation/38400
Anonymous (claudia.sima@****.com) is not covered by the necessary legal agreements in order to proceed.

Thanks

File: tests/plugins/org.polarsys.capella.test.diagram.tools.ju/META-INF/MANIFEST.MF [hunk 1]
```diff
@@ -34,4 +35,4 @@ Export-Package: org.polarsys.capella.test.diagram.tools.ju,
  org.polarsys.capella.test.diagram.tools.ju.xcbd,
  org.polarsys.capella.test.diagram.tools.ju.xdfb,
  org.polarsys.capella.test.diagram.tools.ju.xfbd
-Bundle-Localization: plugin
+Bundle-Localization: plugin
```
Comment (pdulth) [inline tests/plugins/org.polarsys.capella.test.diagram.tools.ju/META-INF/MANIFEST.MF:38]: can you revert this line, blundle-localization is no longer working:

https://ci.eclipse.org/capella/job/capella-product/job/PR-265/1/testReport/junit/org.polarsys.capella.test.platform.ju.testcases/AboutInfoTest/AboutInfoTest/

Comment (georgiana-ecobici) [reply tests/plugins/org.polarsys.capella.test.diagram.tools.ju/META-INF/MANIFEST.MF]: Ok.

**Comment by georgiana-ecobici** [comment]
new pull: https://github.com/eclipse/capella/pull/268

**Pull request closed without merging**